internal/js-dependency-scanner/JsSecurityScanner.iml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file is part of the IntelliJ IDEA project, hence shouldn't be committed. Please ignore `.iml` files in the `.gitignore` (see https://github.com/wso2/carbon-ui-server/blob/master/.gitignore#L57-L104)

Reply:
Resolved.